spec/amp-actions-and-events.md

```diff
@@ -324,7 +322,8 @@ Including: `input[type=radio]`, `input[type=checkbox]`, `input[type=range]`, and
   </tr>
 </table>
 
-## `AMP` target
+## Special targets
```

Comment:
Can you add 1-2 sentences explaining what defines a target as "special"?

For example:

The following are targets provided by the AMP system that have special requirements: